### DISCUSSION TITLE
Some ticks disappear when the X axis is moved.

### DISCUSSION THREAD

**Discussion by dino19981** [chartjs/chartjs-plugin-zoom#865] [Q&A]
Hi. When we increased the X-axis until all labels are visible and then moved the X-axis to the left or right, some of the labels disappear.
Codesandbox: https://codesandbox.io/p/devbox/clever-rosalind-jv5x29?file=%2FApp.tsx%3A177%2C1

https://github.com/chartjs/chartjs-plugin-zoom/assets/81375711/213f8fa8-69f2-4d6c-bbc9-a1d3ccb32674

**Comment by kurkle** [comment]
Add `autoSkip: false` to the x-scale tick options.

<img width="683" alt="image" src="https://github.com/user-attachments/assets/b75c24c8-81fd-42a9-8acc-e2fa7b74a6a4">
